externalAPI/rest/rest_test.go

```diff
@@ -0,0 +1,54 @@
+package rest
+
+import (
+	"net/http"
+	"testing"
+)
+
+func TestAPIClient_buildUrl(t *testing.T) {
+	type fields struct {
+		BaseURL    string
+		Auth       Auth
+		HTTPClient *http.Client
+	}
+	type args struct {
+		method string
+		params []UrlParam
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		want   string
+	}{
+		{
+			name: "check that url parameter values with symbols are fall into url encoded",
```

Comment:
@AndrewKolpakov here

Reply:
"special symbols in URL parameter values are URL-encoded" sounds better to me.

Reply:
How about query parameter names? Should they be encoded too? Path parameters?

Reply:
@AndrewKolpaov yes they will. I propose to add test coverage as separated task

Reply:
fixed